open-build-service/HOWTO-OBS.md

```diff
@@ -0,0 +1,28 @@
+# Open Build Service Steps before release
```

Comment:
Add a one-line preamble to the effect that this is mostly for the notqmail maintainers, but can also be of use to anyone interested in OBS packaging or building their own.

Reply:
This has been addressed in the latest commit #[3eaf213](https://github.com/notqmail/notqmail/pull/76/commits/3eaf2138ce40da0831bfc90852faa8b7c2a2c4ff)